test/Libraries/PackageManagerTests/PackageLoaderTests.cs

```diff
@@ -299,8 +299,16 @@ public void LoadingConflictingCustomNodePackage_AfterPlacingNode_DoesNotGetLoade
             var package2 = Package.FromDirectory(packageDirectory2, CurrentDynamoModel.Logger);
             loader.LoadPackages(new Package[] { package1,package2 });
 
-            // There are 2 packages loaded directly
-            Assert.AreEqual(2, loader.LocalPackages.Count());
```

Comment:
I don't understand this change. If only 2 packages are being loaded we should check for the number to be 2, so this check looks correct as-is.

Reply:
@aparajit-pratap The problem here is that there is no guarantee that only two packages will be loaded. We have tests publishing packages locally which will make these fail. It is passing now on master because the tests mentioned happened to be in failure category and our BRE CI will ignore them properly. But developers running this test locally can reproduce the issue.